qiskit_alice_bob_provider/remote/backend.py

```diff
@@ -70,16 +70,6 @@ def get_translation_stage_plugin(self):
         This function is called before we start the circuit translation,
         and we therefore also use it to inform that we are starting this step.
         """
-        # The first stage to display is Qiskit's inner transpilation step.
```

Comment:
Execute() was removed in Qiskit 0.45+. Now we need to transpile circuits explicitely before running them on a backend.

What this means is that we no longer need to display the circuit is transpiling in the remote backend since these are now much more independant.